From 1a92aacb3cb59f692c8c9725e28a47b2b888d8b7 Mon Sep 17 00:00:00 2001
From: David Wu <david.wu@rock-chips.com>
Date: Fri, 3 Apr 2020 12:27:14 +0800
Subject: [PATCH] ARM: dts: rv112x: Remove i2c scl rising and falling time

These times are all wrong, not real hardware time. For example,
the real rising edge time of i2c0 SCL is about 190ns. In order
to prevent future users from filling in the wrong time, but just
copy, without testing the real time, delete these, and use the
maximum time by default.

Change-Id: I3436b57d9daaeaa8365cb6cf5de7dffbd42db9c9
Signed-off-by: David Wu <david.wu@rock-chips.com>
---
 arch/arm/boot/dts/rv1126-evb-ddr3-v10.dts          | 2 --
 arch/arm/boot/dts/rv1126-iotest-v10.dts            | 2 --
 arch/arm/boot/dts/rv1126-ipc2-ddr3-v10.dts         | 2 --
 arch/arm/boot/dts/rv1126-sphericalipc-ddr3-v10.dts | 2 --
 arch/arm/boot/dts/rv11xx-evb-v10.dtsi              | 2 --
 5 files changed, 10 deletions(-)

diff --git a/arch/arm/boot/dts/rv1126-evb-ddr3-v10.dts b/arch/arm/boot/dts/rv1126-evb-ddr3-v10.dts
index 81a4e3cd500e..c5b8588c54ab 100644
--- a/arch/arm/boot/dts/rv1126-evb-ddr3-v10.dts
+++ b/arch/arm/boot/dts/rv1126-evb-ddr3-v10.dts
@@ -19,8 +19,6 @@
 &i2c1 {
 	status = "okay";
 	clock-frequency = <400000>;
-	i2c-scl-rising-time-ns = <280>;
-	i2c-scl-falling-time-ns = <16>;
 
 	ov4689: ov4689@36 {
 		compatible = "ovti,ov4689";
diff --git a/arch/arm/boot/dts/rv1126-iotest-v10.dts b/arch/arm/boot/dts/rv1126-iotest-v10.dts
index beb3570ad69a..e1f131a0fb8e 100644
--- a/arch/arm/boot/dts/rv1126-iotest-v10.dts
+++ b/arch/arm/boot/dts/rv1126-iotest-v10.dts
@@ -42,8 +42,6 @@
 &i2c0 {
 	status = "okay";
 	clock-frequency = <400000>;
-	i2c-scl-rising-time-ns = <280>;
-	i2c-scl-falling-time-ns = <16>;
 
 	rk809: pmic@20 {
 		compatible = "rockchip,rk809";
diff --git a/arch/arm/boot/dts/rv1126-ipc2-ddr3-v10.dts b/arch/arm/boot/dts/rv1126-ipc2-ddr3-v10.dts
index f09e682354bc..082b436c6bf1 100644
--- a/arch/arm/boot/dts/rv1126-ipc2-ddr3-v10.dts
+++ b/arch/arm/boot/dts/rv1126-ipc2-ddr3-v10.dts
@@ -130,8 +130,6 @@
 &i2c0 {
 	status = "okay";
 	clock-frequency = <400000>;
-	i2c-scl-rising-time-ns = <280>;
-	i2c-scl-falling-time-ns = <16>;
 
 	rk809: pmic@20 {
 		compatible = "rockchip,rk809";
diff --git a/arch/arm/boot/dts/rv1126-sphericalipc-ddr3-v10.dts b/arch/arm/boot/dts/rv1126-sphericalipc-ddr3-v10.dts
index 653b1bd5e78a..a9b21ff6fa1a 100644
--- a/arch/arm/boot/dts/rv1126-sphericalipc-ddr3-v10.dts
+++ b/arch/arm/boot/dts/rv1126-sphericalipc-ddr3-v10.dts
@@ -130,8 +130,6 @@
 &i2c0 {
 	status = "okay";
 	clock-frequency = <400000>;
-	i2c-scl-rising-time-ns = <280>;
-	i2c-scl-falling-time-ns = <16>;
 
 	rk809: pmic@20 {
 		compatible = "rockchip,rk809";
diff --git a/arch/arm/boot/dts/rv11xx-evb-v10.dtsi b/arch/arm/boot/dts/rv11xx-evb-v10.dtsi
index 590ff116f349..343f9e352857 100644
--- a/arch/arm/boot/dts/rv11xx-evb-v10.dtsi
+++ b/arch/arm/boot/dts/rv11xx-evb-v10.dtsi
@@ -497,8 +497,6 @@
 &i2c0 {
 	status = "okay";
 	clock-frequency = <400000>;
-	i2c-scl-rising-time-ns = <280>;
-	i2c-scl-falling-time-ns = <16>;
 
 	rk809: pmic@20 {
 		compatible = "rockchip,rk809";
-- 
2.35.3

